doc/dev/osd_internals/erasure_coding/ecbackend.rst

```diff
@@ -2,18 +2,18 @@
 ECBackend Implementation Strategy
 =================================
 
-Misc initial design notes
+Miscellaneous initial design notes
 =========================
```

Comment:
```suggestion
Miscellaneous initial design notes
==================================
```
the `===` is used for underlining in markdown, can you extend it to cover above text